LICENSE

```diff
@@ -218,13 +218,9 @@ conditions of their respective licenses.
 The Apache Flink project bundles the following files under the MIT License:
 
  - jQuery v2.1.0. (http://jquery.com) - Copyright 2014 jQuery Foundation and other contributors
- - jQuery resize event v1.1 (http://benalman.com/projects/jquery-resize-plugin/) - Copyright (c) 2010 "Cowboy" Ben Alman
```

Comment:
I think the here deleted lines are "transitively" includes, because their contents has been copied into one of the other files we include (by that files' authors). I therefore added them, because I was under the impression that those files were actually not respecting the license conditions properly, and that we should mention these dependencies to be correct.